homeassistant/components/homekit_controller/config_flow.py

```diff
@@ -250,17 +250,6 @@ def __init__(self):
 
         hkid = discovery_props["id"]
 
-        existing = find_existing_host(self.hass, hkid)
```

Comment:
This can't happen any more because of `self._abort_if_unique_id_configured()`, so removing it entirely.